src/numeric/processes.jl

```diff
@@ -55,22 +83,31 @@ function discretize(mvn::MvNormal)
     DiscretizedIIDProcess(x, w)
 end
 
-
-# discrete markov processes
-
-type DiscreteMarkovProcess <: DiscreteProcess
-    transitions::Matrix{Float64}
-    values::Matrix{Float64}
+function rand(mvn::MvNormal, args...)
```

Comment:
Funnily enough, cholesky is the reason why I didn't want to use the Distributions object directly. Reason for that is that if you do `Distributions.MvNormal(zeros(2), zeros(2,2))` (one diagonal zero is enough) you get an error. But we absolutely do want to be able to deal with covariance matrices containing zero eigenvalues, as it comes up quite often. Otherwise, you constantly find yourself defining things like `sigma = 0.000000001` to see what si the perfect foresight version of your model.
Now, if you tell me what kind of wrapper you have in mind, I might understand better the slackness we have to extend/modify the Distributions objects.